src/async-components/views/dialogs/eventindex/ManageEventIndexDialog.tsx

```diff
@@ -26,14 +25,23 @@ import {formatBytes, formatCountLong} from "../../../../utils/FormattingUtils";
 import EventIndexPeg from "../../../../indexing/EventIndexPeg";
 import {SettingLevel} from "../../../../settings/SettingLevel";
 
+interface IProps {
+    onFinished: (boolean) => void;
```

Comment:
this defined a parameter of name `boolean` but with no type
![image](https://user-images.githubusercontent.com/2403652/116108745-19100c80-a6ac-11eb-9584-bff030e0a7fa.png)